app/questionnaire/placeholder_transforms.py

```diff
@@ -75,10 +79,9 @@ def format_possessive(self, string_to_format):
 
         return string_to_format
 
-    @staticmethod
-    def format_number(number):
+    def format_number(self, number):
         if number or number == 0:
-            return numbers.format_decimal(number, locale=PlaceholderTransforms.locale)
+            return numbers.format_decimal(number, locale=self.locale)
```

Comment:
```suggestion
            return format_decimal(number, locale=self.locale)
```

This would be more consistent with `format_currency` and `format_datetime`, which are imported explicitly from the Babel numbers package.